src/analysisd/format/json_extended.c

```diff
@@ -194,6 +194,47 @@ void W_JSON_ParseGroups(cJSON* root, const Eventinfo* lf)
         }
         token = strtok(0, delim);
     }
+
+    //Add SCA compliance groups
+    cJSON *data = cJSON_GetObjectItem(root,"data");
+    if(data){
+        cJSON *sca = cJSON_GetObjectItem(data,"sca");
+        if(sca){
+            cJSON *check = cJSON_GetObjectItem(sca,"check");
+            if(check){
+                cJSON *compliances = cJSON_GetObjectItem(check,"compliance");
+                cJSON *compliance;
+                cJSON_ArrayForEach(compliance,compliances){
+                    add_SCA_groups(rule, compliance->string, compliance->valuestring);
+                }
+            }
+        }
+    }
+}
+
+void add_SCA_groups(cJSON *rule, char* compliance, char* value){
+    char *aux;
+    os_strdup(value, aux);
+    cJSON *group = cJSON_GetObjectItem(rule, compliance);
+    if(!group){
+        group = cJSON_CreateArray();
+        cJSON_AddItemToObject(rule, compliance, group);
+    }
+    char *token;
+    char *state;
+    token = strtok_r(aux, ",", &state);
+    cJSON_AddItemToArray(group, cJSON_CreateString(token));
+    if(strlen(state) > 0){
+        if(state[0] == ' ')
```

Comment:
Here we only trim the first whitespace, so, for fields like `  1.1` with two spaces, only the first one is removed.